web/conf.py

```diff
@@ -1210,6 +1214,11 @@
 # it appears on the navigation bar:
 #
 # SEARCH_FORM = """
+SEARCH_FORM = """
+<form class="navbar-search pull-right" action="/search/" role="search">
+    <input type="text" class="form-control" id="tipue_search_input" name="q" placeholder="Search" autocomplete="off">
+</form>
+"""
```

Comment:
move up one line so this definition of `SEARCH_FORM` is not in the middle of the commented out definition of `SEARCH_FORM`